ingestion/ingestion/filesystem.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import annotations
+
+import json
+import logging
+from pathlib import Path
+from typing import Any, get_args
+
+from ingestion.schema import DataAnnotationEntry, DataContainer
+
+logger = logging.getLogger(__name__)
+
+_NEURONS_FILE = Path("neurons.json")
```

Comment:
It's not necessary to define them as `Path(...)` instance as they are only used as end of segments for a Path later in the file.